packages/Frontend/src/components/Authenticate.jsx

```diff
@@ -54,6 +69,8 @@ export default function Authenticate({
               <Button color="violet" fluid>{SIGN_IN}</Button>
             </Link>
           }
+          <Button color="blue" onClick={() => authenticate('google')} fluid>Sign In With Google</Button>
+          <Button color="blue" onClick={() => authenticate('facebook')} fluid>Sign In With Facebook</Button>
```

Comment:
These are both ugly buttons that I copied from the other button, we should switch them out with Google and Facebook sign in images when we have time. I changed the authenticate function to take a string of what provider we want to authenticate with, which you'll see later.